main.js

```diff
@@ -82,7 +100,7 @@ function setupMenu() {
   Menu.setApplicationMenu(Menu.buildFromTemplate(template));
 }
 
-function initApp() {
+function initUiWin() {
```

Comment:
nit: rename to `initUI`? `Win` can be a confusing acronym for windows OS

Reply:
done